### PR TITLE
Fixes #5447

### LINKED ISSUE #5447: Incorrect documentation for Service Leaf Certificate API
Consul version: v1.4.3

I have, for the last 3 hours been trying to work out how to generate a certificate for a native application to use Consul Connect based on the documentation [here](https://www.consul.io/api/agent/connect.html#service-leaf-certificate), where it describes the `Service Leaf Certificate` endpoint as:-

`/agent/connect/ca/leaf/:service`

and then gives an example as:-

`curl http://127.0.0.1:8500/v1/connect/ca/leaf/web` 

The problem is, neither of these are correct, I realised this after re-reading the section on [`Accepting Inbound Connections`](https://www.consul.io/docs/connect/proxies/integrate.html#accepting-inbound-connections) where it has a link to the HTTP API but has the link text as:-

`/v1/agent/connect/ca/leaf/`

Note that this is actually a combination of both the API method definition and ALSO the example and the actual endpoint is:-

`http://127.0.0.1:8500/v1/agent/connect/ca/leaf/web`

which does work!

Incidentally, it looks like the .../ca/roots documentation in the HTTP API page is also wrong as it is also documented similar to the leaf API endpoint but amusingly works for both and contributes towards confusion as to why the leaf endpoint returns a 404.

`curl -iv http://127.0.0.1:8500/v1/connect/ca/roots`

and

`curl -iv http://127.0.0.1:8500/v1/agent/connect/ca/roots`

### PR DESCRIPTION
**Pull request by andye2004**
Some further updates to the documentation to update some areas missed in the previous PR #5715

### CODE REVIEW

File: website/source/api/agent/connect.html.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |-
 
 # Connect - Agent HTTP API
 
-The `/agent/connect` endpoints interact with [Connect](/docs/connect/index.html)
+The `v1/agent/connect` endpoints interact with [Connect](/docs/connect/index.html)
```
Comment (banks) [inline website/source/api/agent/connect.html.md]: Hey thanks for this although the version prefix is intentioally omitted on all out API docs in theory. We mention this in: https://www.consul.io/api/index.html#version-prefix.

For example check any other API docs page - none of the URLs have /v1/ prefix. https://www.consul.io/api/catalog.html. We could potentially update the wording on the link above to make that more obvious but I don't want to change that in this file and make it inconsistent with all other docs.

Comment (banks) [reply website/source/api/agent/connect.html.md]: image

Random example.

File: website/source/api/agent/connect.html.md [hunk 2]
```diff
@@ -259,7 +259,7 @@ blocking](/api/index.html#hash-based-blocking-queries) only.
 
 ```text
 $ curl \
-   http://127.0.0.1:8500/v1/connect/proxy/web-proxy
+   http://127.0.0.1:8500/v1/agent/connect/proxy/web-proxy
```
Comment (banks) [inline website/source/api/agent/connect.html.md]: This is a legit fix that would be good to merge, thanks!

File: website/source/api/agent/connect.html.md (hunk 1 above)
Comment (banks) [reply website/source/api/agent/connect.html.md]: If you want to update this PR so the commit retains your autorship I'll merge the curl fix right away but it's also trivial so if you happy to do it if you don't have time!

File: website/source/api/agent/connect.html.md (hunk 2 above)
Comment (andye2004) [reply website/source/api/agent/connect.html.md]: Authorship really doesn't worry me Paul, please go ahead and just make the change and I'll close the PR.

File: website/source/api/agent/connect.html.md (hunk 1 above)
Comment (andye2004) [reply website/source/api/agent/connect.html.md]: > Hey thanks for this although the version prefix is intentioally omitted on all out API docs in theory. We mention this in: https://www.consul.io/api/index.html#version-prefix.
> 
> For example check any other API docs page - none of the URLs have /v1/ prefix. https://www.consul.io/api/catalog.html. We could potentially update the wording on the link above to make that more obvious but I don't want to change that in this file and make it inconsistent with all other docs.

Thanks for pointing this one out, I shall remember in future :)

**Comment by andye2004** [comment]
@banks will make the required changes directly to the hashicorp codebase, therefore closing this particular PR.

**Pull request closed without merging**